addons/popochiu/migration/migration/popochiu_migration.gd

```diff
@@ -60,7 +64,9 @@ func get_migration_name() -> String:
 ## Returns [true] if the current Popochiu migration version is newer than the user's migration
 ## version, which means a migration is needed.
 func is_migration_needed() -> bool:
-	return _version > PopochiuMigrationHelper.get_user_migration_version()
+	return (
+		_version > PopochiuMigrationHelper.get_user_migration_version() and _is_migration_needed()
```

Comment:
Related to commet at line 35: either you add a `const FORCE_MIGRATION = true` atop the file, then

```gdscript
		_version > PopochiuMigrationHelper.get_user_migration_version() and FORCE_MIGRATION
```

Or you just

```gdscript
		_version > PopochiuMigrationHelper.get_user_migration_version() and true
```

for this time.
Am I missing something?

Reply:
Take a look to the answer for the comment above 😉 .